### DISCUSSION TITLE
Admin View

### DISCUSSION THREAD

**Discussion by Prajwalprakash3722** [IEEE-RVCE/GateKeeper#59] [Ideas]
What all do we need in admin view

**Comment by SundarakrishnanN** [comment]
Need Different sections: User section ,Society Section. Let there be a possibility to add more sections later on..
### User Section
Different options to view/edit users.. add users and also bulk create users(lets keeep it for later).
### Society Section
Different options to view/edit societies.add society.

**Reply by Prajwalprakash3722** [reply]
> bulk create users

Allow the frontend to accept a CSV file, convert it to JSON, and either send a series of requests to create user or send the data to a new backend route that will handle the bulk user creation. 

https://github.com/IEEE-RVCE/GateKeeper/issues/24
